pkg/logging/config.go

```diff
@@ -17,14 +17,17 @@ limitations under the License.
 package logging
 
 import (
+	"fmt"
+	"os"
+
 	"go.uber.org/zap"
 	corev1 "k8s.io/api/core/v1"
 
 	"github.com/knative/pkg/logging"
 )
 
 const (
-	ConfigName = "config-logging"
+	ConfigMapNameEnv = "CONFIG_LOGGING_NAME"
```

Comment:
nit: for single constant use one line

i.e

`const ConfigMapNameEnv = "CONFIG_LOGGING_NAME"`

Reply:
Done